Formula/virgil.rb

```diff
@@ -0,0 +1,25 @@
+class Virgil < Formula
+  desc "CLI tool to manage your Virgil account and applications"
+  homepage "https://github.com/VirgilSecurity/virgil-cli"
+  url "https://github.com/VirgilSecurity/virgil-cli.git",
+     :tag      => "v5.0.1",
+     :revision => "03ddee65e9cecbdd7ef55b1285c0ca70758d6d40"
+  head "https://github.com/VirgilSecurity/virgil-cli.git"
+  depends_on "dep" => :build
```

Comment:
Please leave some whitespace here for readability.
```suggestion

    depends_on "dep" => :build
```